test/quantization/test_quantize_script.py

```diff
@@ -1116,10 +1116,12 @@ class TestQuantizeScriptPTSQOps(QuantizationTestCase):
     """ Test graph mode post training static quantization works
     for individual ops end to end.
     """
-    def _test_op_impl(self, module, data, quantized_op):
+    def _test_op_impl(self, module, data, quantized_op, debug=False):
         qconfig_dict = {'': get_default_qconfig(torch.backends.quantized.engine)}
         model = torch.jit.script(module).eval()
         model = quantize_script(model, qconfig_dict, _test_only_eval_fn, [data], inplace=False)
+        if debug:
```

Comment:
Do you want to have this be part of the final code?

Reply:
yeah I think we can have a debug option